### ISSUE TITLE
Changing colors causes the shade function to no longer work.

### ISSUE THREAD

**Issue by jmitchell89** [luckylooke/phaser-islandjs-plugin#6]
I start to get these bright pinks/purples and bright blues when I attempt to use other colours.

**Comment by luckylooke** [comment]
I can hardly imagine.. please reproduce your problem somewhere public so I can see it.. like codepen, jsfiddle, etc.. ;)

**Comment by jmitchell89** [comment]
Sure thing here is an example:

https://codepen.io/anon/pen/JyOJKj

There are two color schemes. The first one is not that bad just a couple of weird spots, but the 2nd one.... well you'll be able to see... lol

Also if you turn off the shading by using 0 for the shading value you'll see that the weird colors disappear.

**Comment by luckylooke** [comment]
Yes, it looks creepy :D I didn't face this issue before. I will look at it more when I will have time for it.. but I will not lie to you, this issue does not have high priority for me now. I have really much more to do, so better for you will be to debug this issue and make PR. Thanks for understanding.
